e2e/testcafe-devextreme/tests/editors/list/grouping.ts

```diff
@@ -53,6 +54,37 @@ test('Grouped list appearance', async (t) => {
   },
 }));
 
+test('Grouped List last item of last group should have margin-bottom = 4px', async(t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  
+  const items = Selector('.dx-item');
```

Comment:
Let's use List model for this.

Reply:
[640d4e0](https://github.com/DevExpress/DevExtreme/pull/27093/commits/640d4e0d7b58662795cac821b7a1415c4d807f6d)